templates/custom/nodes/node--news--teaser.html.twig

```diff
@@ -12,5 +12,18 @@
 
   {% set image_ratio = '1:1' %}
   {# twigcs use-var image_ratio #}
-  {{ parent() }}
+  {% if node.hasField('field_teaser_image') %}
+    {% if image_ratio is not defined %}
+      {% set image_ratio = '8:5' %}
```

Comment:
Is deze niet altijd set op regel 13?

Reply:
Klopt. Aangepast zodat het nu 1:1 is standaard